docs/YIELDGENERATOR.md

```diff
@@ -24,14 +24,14 @@ Open one of the two built-in yield generators: `yield-generator-basic.py` or `yg
 	txfee = 1000
 	cjfee_a = 2000
 	cjfee_r = '0.0002' # 0.02% fee
-	ordertype = 'swreloffer'
+	ordertype = 'sw0reloffer'
```

Comment:
Oh we need to change this now. Just dropping a note, I can do it.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/656/commits/1d21f304af261a74f5c0dd024f79def113b551b0